.env.example

```diff
@@ -1,17 +1,25 @@
-# RPC Endpoints
```

Comment:
*Arbitrary comment location*

* Any reason we shouldn't check in the `package-lock.json`?
* What would you think about using [volta]() in the `package.json` to ensure consistent development environments amongst different devs + prod deployment

Reply:
i can make an issue for this